src/controllers/api.controller.ts

```diff
@@ -25,6 +26,28 @@ export const claimPost = async (
         ...rawClaim
       }
     });
+
+    // fetch first claim on same subject and then update the new with this format
+    // https:/live.linkedtrust.us/claims/${THE_REALTED_claimId}
+    const relatedClaim = await prisma.claim.findFirst({
+      where: {
+        subject: claim.subject,
+        id: {
+          not: claim.id
+        }
+      }
+    });
+
+    if (relatedClaim) {
+      claim = await prisma.claim.update({
```

Comment:
NO we do NOT want to update claims in the database, they have been signed!  this is not what i asked actually i wanted you only to use makeClaimSubjectURL when FIRST making the claim

Do NOT update Claim in the database please in the code.  if we have to do one time migration we will and re-sign.

Reply:
Claim is like the raw signal think of it like kafka event

Node and Edge is derived and can be updated

Claim will be signed with DID when it is created it should never be updated